bin/real-wazo-upgrade

```diff
@@ -107,6 +111,15 @@ upgrade() {
 
 upgrade_dahdi_linux_modules(){
 	VERSION=$1
+
+	if ! is_package_installed "dahdi"; then
+		return
+	fi
+
+	if is_package_installed "dahdi-linux-modules-$(uname -r)"; then
```

Comment:
why this check? If we do a `wazo-upgrade` and the kernel is updated, then on the next restart, dahdi will be broken